controllers/applicationsController/deleteSingleApplication.js

```diff
@@ -0,0 +1,31 @@
+const CustomError = require("../../utils/customError");
+const responseHandler = require("../../utils/responseHandler");
+const Applications = require("../../models/applications");
+
+const deleteSingleApplication = async (req, res, next) => {
+  const applicationId = req.params.applicationId;
+  try {
+    const application = await Applications.findById(applicationId);
```

Comment:
You need to confirm that the application belongs to the same organization as the admin requesting this delete operation. `organizationId` of the user is available on `req.token`. `organizationId` is also a property of the application. Compare these two prevent an admin from accidentally deleting an application belonging to a different organization